packages/headless/esbuild.mjs

```diff
@@ -131,39 +141,49 @@ function buildBrowserConfig(options) {
     ],
     ...options,
   });
+  outputMetafile(`browser.${options.format}`, outDir, out.metafile);
```

Comment:
Similarly to the Stencil definitions file, I think it may be useful to include the metafile in Git/PRs. It looks like it would make it easy to identify changes in the bundle at a pretty low effort cost.

Reply:
I want to work on this at a later stage: But yes, eventually I want to add reports directly in PR to see changes. 

I've explored some tools that will allow us to do this (eg: https://esbuild.github.io/analyze/)